odxtools/diagcomm.py

```diff
@@ -61,8 +61,8 @@ class DiagComm(IdentifiableElement):
     audience: Optional[Audience]
     protocol_snrefs: List[str]
     related_diag_comm_refs: List[RelatedDiagCommRef]
-    pre_condition_state_refs: Iterable[OdxLinkRef]
-    state_transition_refs: Iterable[OdxLinkRef]
+    pre_condition_state_refs: List[OdxLinkRef]
+    state_transition_refs: List[OdxLinkRef]
```

Comment:
```suggestion
    state_transition_refs: list[OdxLinkRef]
```

Reply:
yes. once we can rely on python 3.10, cf #286. The blocker is Ubuntu 20.04, which goes EOL in [April](https://ubuntu.com/about/release-cycle). I think we should be patient enough to wait the remaining three months...